tests/test_cli.py

```diff
@@ -0,0 +1,92 @@
+import os
+from pathlib import Path
+from sabres.cli import main
+from sabres import merge_sabres
+import pytest
+
+
+
+@pytest.mark.parametrize(
+    "options,expected", 
+    [
+        (["--help"], "show this help message and exit"),
+        (["-h"], "show this help message and exit"),
+        (["--version"], "SABRes v"),
+])
+def test_helpers(capsys, options, expected):
+    try:
+        main(options)
+    except SystemExit:
+        pass
+    
+    result = capsys.readouterr().out
+    assert expected in result
+
+
+@pytest.mark.parametrize(
+    "options,expected", 
+    [
+        (["--vcall"], "argument --vcall/-v: expected one argument"),
+        ([], "the following arguments are required: --vcall/-v, --input/-i"),
+        (["--vcall", "ivar"], "the following arguments are required: --input/-i"),
+        (["--vcall", "ivar", "--outdir", 'outdir'], "the following arguments are required: --input/-i"),
+])
+def test_missing_args(capsys, options, expected):
+    try:
+        main(options)
+    except SystemExit:
```

Comment:
Also pytest has a special `pytest.rasies()` for capturing expected exceptions, https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions
don't except pass.